.gitlab-ci.yml

```diff
@@ -27,44 +27,41 @@ variables:
 
 include:
   - project: "Northern.tech/Mender/mendertesting"
-    file: ".gitlab-ci-check-commits.yml"
-  - project: "Northern.tech/Mender/mendertesting"
-    file: ".gitlab-ci-github-status-updates.yml"
+    file:
+      - ".gitlab-ci-check-commits.yml"
+      - ".gitlab-ci-github-status-updates.yml"
+  - local: "/frontend/pipeline.yml"
 
 stages:
+  - pre-test
   - build
   - test
+  - e2e-test
```

Comment:
Isn't end-to-end tests a test?

Reply:
yep, related to the changed execution order from the general pipeline setup... will adjust.